bee-node/Cargo.toml

```diff
@@ -10,6 +10,12 @@ license = "Apache-2.0"
 keywords = [ "iota", "tangle", "bee", "framework", "node" ]
 homepage = "https://www.iota.org"
 
+[package.metadata.docs.rs]
+# To build locally:
+# RUSTDOCFLAGS="--cfg doc_cfg" cargo +nightly doc --all-features --no-deps --open
+all-features = true
+rustdoc-args = [ "--cfg", "doc_cfg" ]
+
```

Comment:
I'm afraid we can't get rid of these in somewhat unrelated crates, right? Or do they make sense anyways, independent of the changes to `bee-message`?

Reply:
I also find that also very strange. 🤔